### ISSUE TITLE
paddleocr的精度不够高，发现一些gt错误的case

### ISSUE THREAD

**Issue by moyans** [ShuoZhang2003/DT-VQA#2]

**Comment by cooleel** [comment]
>paddleocr的精度不够高，发现一些gt错误的case

Could you please provide some examples on this? Thanks. @moyans

**Comment by moyans** [comment]
I am sorry for the late reply。 such as:   train_DTProdCap_train_val.json
`    "id": "./image/DTProdCap/train_val/TB1vribiyAnBKNjSZFvXXaTKXXa_!!0-item_pic.jpg",
    "conversations": [
      {
        "from": "user",
        "value": "<img>./image/DTProdCap/train_val/TB1vribiyAnBKNjSZFvXXaTKXXa_!!0-item_pic.jpg</img> What is the product name? Answer: "
      },
      {
        "from": "assistant",
        "value": "法拉榭"
      },`
      
![TB1vribiyAnBKNjSZFvXXaTKXXa_!!0-item_pic](https://github.com/user-attachments/assets/f45f7394-6322-4671-82bc-07022fd65aac)

**Comment by cooleel** [comment]
> I am sorry for the late reply。 such as: train_DTProdCap_train_val.json ` "id": "./image/DTProdCap/train_val/TB1vribiyAnBKNjSZFvXXaTKXXa_!!0-item_pic.jpg", "conversations": [ { "from": "user", "value": "<img>./image/DTProdCap/train_val/TB1vribiyAnBKNjSZFvXXaTKXXa_!!0-item_pic.jpg</img> What is the product name? Answer: " }, { "from": "assistant", "value": "法拉榭" },`
> 
> ![TB1vribiyAnBKNjSZFvXXaTKXXa_!!0-item_pic](https://private-user-images.githubusercontent.com/13639445/353416040-f45f7394-6322-4671-82bc-07022fd65aac.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM3Mzk2NzQsIm5iZiI6MTcyMzczOTM3NCwicGF0aCI6Ii8xMzYzOTQ0NS8zNTM0MTYwNDAtZjQ1ZjczOTQtNjMyMi00NjcxLTgyYmMtMDcwMjJmZDY1YWFjLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE1VDE2MjkzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU5NGJiNTFiOTljNzFiMTA3YjhhNmVkMjQxOWI4ZmQwYmQxYzgxN2MzOGZjZGU5YjEwODIyNzQ5NTc3NmY5NTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.VPtY76aBvVg6t5GztIKBiEZeYZZqghxeLKIOD5hdOSM)

Thanks a lot for the response, do you have any idea how many examples have this issue?

**Comment by ShuoZhang2003** [comment]
We apologize for the inconvenience. Since the annotations in the training set were generated automatically by the model, there are some errors due to limitations in the capabilities of both PaddleOCR and the multimodal large model. These errors are unfortunately unavoidable. For the test set, we conducted manual screening to ensure the accuracy of the testing.
